cytosnake/utils/cyto_paths.py

```diff
@@ -23,10 +25,14 @@ def get_meta_path() -> Path:
     """
 
     # getting project directory
-    proj_dir = find_project_dir()
-    if not proj_dir.exists():
+    proj_dir = find_project_dir() / ".cytosnake"
+    if proj_dir is None:
         raise FileNotFoundError("Current directory is not a project folder")
 
+    # # checking if it is a valid path
+    # if not is_cytosnake_dir():
```

Comment:
Should this commented-out code remain in this file? If not, consider removing.

Reply:
removed it!